packages/api/src/controllers/stream.ts

```diff
@@ -841,12 +842,21 @@ app.get("/:id", authorizer({}), async (req, res) => {
 
 // returns stream by steamKey
 app.get("/playback/:playbackId", authorizer({}), async (req, res) => {
-  const {
-    data: [stream],
-  } = await req.store.queryObjects<DBStream>({
-    kind: "stream",
-    query: { playbackId: req.params.playbackId },
-  });
+  const stream = await cache.getOrSet(
+    `strm-stream-by-playback-${req.params.playbackId}`,
+    async () => {
+      const {
+        data: [stream],
+      } = await req.store.queryObjects<DBStream>({
+        kind: "stream",
+        query: { playbackId: req.params.playbackId },
+      });
+
+      return stream;
+    },
+    5
```

Comment:
Is this the TTL? If so, any reason to have it so short? Is it gonna have any gain at all?

AFAIK this API is not widely used, and the cache is local to the API instance. So a cache of 5s would hardly ever get any hits.

I'd go with 60s at least.

Reply:
The issue is explained here https://discord.com/channels/423160867534929930/1102968315137228900/1246573851358265384

It's true that it's not widely used, but go-api-client is pointing to that via `GetStreamByPlaybackId` and this endpoint is getting flooded by calls due to a `lockPull` bug that Rafal is fixing.
We're receiving around 4k req/s to this endpoint during the fails, so this is to mitigate the amount of queries going into postgres. We probably want to call the /stream endpoint instead from go-api-client, but that's without cache too.

5s because the active state is needed, but we can increase, maybe to 10 or 15, I think it's a @leszko call depending on how much the state of the stream needs to be updated.

Reply:
Thanks for explaining. This would have been super helpful context to add on the PR description 😄 

5s sounds fine if it's only coming in surges.